test/external/dataplane/connectivity.go

```diff
@@ -228,3 +242,16 @@ func getExternalClusterIndex(names []string) framework.ClusterIndex {
 	// TODO: consider right error handling
 	return framework.ClusterIndex(0)
```

Comment:
Instead of returning framework.ClusterIndex(0) which could be wrong and misleading when the test-case fails. I think it might be better to fail the test-case. What do you say?

Reply:
Modified to return error as the second return value and handle the error in caller.